tasks/main.yml

```diff
@@ -25,3 +25,10 @@
   template:
     src: etc/squid/conf.d/dummy.conf.j2
     dest: "{{ squid_root_dir }}/conf.d/dummy.conf"
+
+- name: Setup domain lists (black and white)
+  include: config_squid_domain_lists.yml
+  when: >
+    squid_domain_lists_dir != "" and
+    (squid_domain_blacklist_file is defined or
```

Comment:
The filenames are always defined.
It's probably better to change this to:

( squid_domain_blacklist | length>0 ) or
( squid_domain_whitelist | length>0 )

Reply:
Good catch, updated the code.